.babelrc

```diff
@@ -1,7 +1,9 @@
 {
   "presets": [
-    "es2015",
-    "flow",
-    "stage-2",
```

Comment:
isn't stage2 what has the arrow syntax & some other useful features?

Reply:
The preferred method is the `env` preset, which figures out the appropriate presets based on your code